src/components/CodeEditor/index.jsx

```diff
@@ -18,7 +18,9 @@ import './json-lint';
     width: '100%',
   },
 })
-/** Render an editor */
+/** Render an editor. <br>
+ * * The following component depends on `Material-UI`.
+ */
```

Comment:
```diff
diff --git a/src/components/CodeEditor/index.jsx b/src/components/CodeEditor/index.jsx
index 4b4fdac..4658574 100644
--- a/src/components/CodeEditor/index.jsx
+++ b/src/components/CodeEditor/index.jsx
@@ -19,7 +19,8 @@ import './json-lint';
   },
 })
 /** Render an editor. <br>
- * * The following component depends on `Material-UI`.
+ *
+ * _Note: [material-ui](https://material-ui.com/) is a peer-dependency_
  */
 export default class CodeEditor extends Component {
   static propTypes = {
```

Let's do the same change for the other files that require material-ui.

![Screen Shot 2019-10-14 at 10 12 00 AM](https://user-images.githubusercontent.com/3766511/66757875-19024000-ee6b-11e9-8d34-c40ce720d2b7.png)

Reply:
Looks nice